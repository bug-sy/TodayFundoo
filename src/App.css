.sign-up-wrapper{
height:800px;
width:'100%';
display:flex;
background-color:lightcyan;
justify-content:center;
}

.sign_up_content{
  height:800px;
  width:'100%';
   background:red; 
  display:flex;
  justify-content:center;
}

.icon{
  height: 450px;
  display : flex ;
  width:700px;
  flex-direction: row;
  justify-content:center;
  margin-top: 200px;
  border: 1px solid black;
}

.main_page{
height:800px;
width:'100%';
display: flex;
justify-content: center;
}

.sign_up{
  justify-content:'space-between';
  height: 450px;
  display : flex ;
  width:700px;
  flex-direction: column;
  align-items: center;
  margin-top: 200px;
  border: 1px solid black;
  margin-left:500px;

}

.firstdiv{
height:300px;
margin-left: 30px;
}

.Email{
  width:340px;
}

.sign-up-form input {
display: flex;
justify-content:'space-between';
align-items: center;
border-style: outset;
width : 1200px;
height : 1000px;
margin-left: 600px;
border: 1px solid lightgray; 
flex-direction: row; 
}

.sign_up_button{
  margin: 5px;
  margin-left: 95px;
}

.sign_in_button{
  margin: 10px;
  margin-left: 130px;
}

.image{
  margin: 5px;
  height :200px;
  width :200px;
  margin-top: 50px;
}


@media only screen and (max-width: 600px) {
.image{
display: none;
}

.firstdiv{
height:300px;
}

.icon{
  height: 450px;
  display : flex ;
  width:700px;
  flex-direction: row;
  justify-content:center;
  margin-top: 75px;
  border:0  
}

.firstname{
  width:125px;
}

.lastname{
  width:125px;
}

.Email{
  width:270px;
}

.sign_up_button{
  margin: 5px;
  margin-left: 45px;
}

.sign-up-wrapper{
  border: 1px solid black;
  height:600px;
  width:400px;
  background-color: lawngreen;
  margin-left: 25px;
}
}

.divisionforNotes{
  height:150px;
  width:400px;
  margin-left: 480px;
  margin-top: 150px;
}



