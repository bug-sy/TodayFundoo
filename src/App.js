import React from 'react';
import Reminder from './Reminder/ComponentOfReminder/ReminderDataFetching'
import './App.css';
import LabelMapping from '../src/components/LabelComponent/LabelMapping.js'
import EditLabelDialog from '../src/components/Label/EditLabelDialog.js'
import { Provider } from 'react-redux'
import store from './components/UsingRedux/store'
import { BrowserRouter, Route } from 'react-router-dom'
import PageNotFound from './components/Pages/PageNotFound'
import Dashboard from './components/Dashboard'
import Parsing from './components/Pages/Parsing'
import UserSignupmain from './components/UserSingnupmain'
import UserSigninmain from './components/UserSigninmain'
import Vert from '../src/components/MMainnote.js'
import Archive from './components/Archive'
import Trash from './components/Trash'
import AddLabel from './components/Label/NoteCardLabel'
import Stand from '../src/components/standalone/DialogueList.js'

class App extends React.Component{

  render(){
    return (
      <Provider store={store}>
    <div className="App">
      <BrowserRouter >
      
          <Route exact path="/Vert" component={Vert} />
          <Route exact path="/Dashboard/Parsing" component={Parsing} />
          <Route path="/EditLabelDialog" component={EditLabelDialog}/>
          <Route path="/Signin" component={UserSigninmain} />
         
          <Route path="/Signup" component={UserSignupmain} />
          <Route path="/Dashboard/Trash" component={Trash} />
          <Route path="/Dashboard/Archive" component={Archive} />
          <Route path="/Dashboard/Reminder" component={Reminder}/>
          <Route exact path="/Vert " component={Vert} />
          <Route  path="/Dashboard" component={Dashboard}/>
            <Route path="/Dashboard/LabelMapping" component={LabelMapping}/>
          <Route path="/Stand" component={Stand}/>
          <Route exact path="/" component={PageNotFound} />
      </BrowserRouter>
    </div>
      </Provider>
            );
  }
}

export default App;
