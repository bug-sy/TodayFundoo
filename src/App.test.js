import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('renders learn react link', () => {
  const { container } = render(<App />);
  container.querySelector('.my-class')
  expect(container.firstChild).toMatchSnapshot()
});
