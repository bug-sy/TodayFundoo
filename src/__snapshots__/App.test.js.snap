// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="App"
>
  <div>
    <h3>
      Page NOT Found
    </h3>
  </div>
</div>
`;
